examples/with-server/src/server/start.ts

```diff
@@ -10,11 +10,11 @@ export interface AppState {}
 createServer({
     log,
     // Use the middleware hook to register your own middleware
-    middlewareHook: app => {
+    middlewareHook: (app: { get: (arg0: string, arg1: (_: any, res: any) => any) => void }) => {
```

Comment:
This shouldn't be necessary&mdash;typescript should be able to infer the type of `app`.

Try updating the references in examples/with-server/tsconfig.json to include `{ "path": "../../packages/server" }`&mdash;there's a working example in examples/ssr-with-sass/tsconfig.json